lib/cli/src/commands/wasmer_create_exe_main.c

```diff
@@ -14,21 +10,17 @@ extern "C" {
 // TODO: make this define templated so that the Rust code can toggle it on/off
 #define WASI
 
-#ifdef __cplusplus
-}
-#endif
-
-void print_wasmer_error()
-{
+static void print_wasmer_error() {
   int error_len = wasmer_last_error_length();
   printf("Error len: `%d`\n", error_len);
-  char* error_str = (char*) malloc(error_len);
+  char *error_str = (char *)malloc(error_len);
```

Comment:
I will never believe that `char *name` is more readable or accurate than `char* name` 😆

Reply:
`char x, *y, f()` is a char, a pointer-to-char and a function that returns a char. What a language! :laughing: